omxplayer.cpp

```diff
@@ -453,7 +453,7 @@ static int get_mem_gpu(void)
 
 static void blank_background(uint32_t rgba)
 {
-  if (!rgba)
+  if (!(rgba & 0xff000000))
     return;
```

Comment:
Add at least some comment to this line why you prefer (/do) this. For someone not familiar with this code, this could be confusing as the naming variable 'rgba' doesn't help your intention. And yes, basically you are right...